python/mujoco/usd/exporter.py

```diff
@@ -413,19 +401,47 @@ def add_camera(
       self,
       pos: List[float],
       rotation_xyz: List[float],
-      objid: Optional[int] = 1,
+      obj_name: Optional[str] = "camera_1",
   ):
-    new_camera = component_module.USDCamera(
+    new_camera = camera_module.USDCamera(
         stage=self.stage, obj_name=str(objid))
 
     r = scipy.spatial.transform.Rotation.from_euler(
         "xyz", rotation_xyz, degrees=True)
     new_camera.update(cam_pos=np.array(pos), cam_mat=r.as_matrix(), frame=0)
 
   def save_scene(self, filetype: str = "usd"):
+    assert filetype in ["usd", "usda", "usdc"]
     self.stage.SetEndTimeCode(self.frame_count)
+
+    # post-processing for visibility of geoms in scene
+    for _, geom_ref in self.geom_refs.items():
+      geom_ref.update_visibility(False, geom_ref.last_visible_frame+1)
+
     self.stage.Export(
         f"{self.output_directory_root}/{self.output_directory_name}/frames/frame_{self.frame_count}_.{filetype}"
```

Comment:
Hi, is this a typo to have an additional `_` after `{self.frame_count}`?

Reply:
I remember this was intentional prior to when we were able to render a full trajectory. However, now, I do not believe there is a reason. I can remove this.